tests/test_utils.py

```diff
@@ -51,3 +51,18 @@ def test_filter_dict_by_key():
     fd = utils.filter_dict_by_key(d, keys)
     assert len(fd) == 1
     assert all([k in keys for k in fd])
+
+
+def test_is_number_int():
+    """Test that is_number returns True for integers."""
+    assert utils.is_number('2') is True
+
+
+def test_is_number_float():
```

Comment:
This actually does not work for numbers formatted with a thousand separator.

```
ipdb> float('1,000,000.0')
*** ValueError: invalid literal for float(): 1,000,000.0
```

And because `align_decimals` now comes after `format_numbers`, aligning decimals stopped working if a thousand separator is specified in `pgclirc`.